src/ui/components/navigation/navigationcomponent.js

```diff
@@ -379,6 +385,19 @@ export default class NavigationComponent extends Component {
       }
     }
 
+    if (this._parentUrl) {
+      const parentUrlWithoutParams = this._parentUrl.split('?')[0];
+      const urlParser = document.createElement('a');
```

Comment:
Is it necessary that we create an anchor tag here just to update the `tabs`? That seems like a bit of a code smell. Every time this component is re-rendered, will more and more of the anchor tags get appended to the DOM?

Reply:
the use of the anchor tag here is not to append to DOM but to help parse pathname. This will not be append to the dom. From looking online, I found this to be a simple alternative to manually parse the url ourselves, which would be tricky since we have to deal with relative and absolute urls and their different formats as well. Alternatively we could use URL function but we would need a polyfill for IE11.